force-app/main/default/classes/BDI_SettingsUI_TEST.cls

```diff
@@ -76,14 +76,15 @@ private with sharing class BDI_SettingsUI_TEST {
         Boolean containsImportedField = false;
 
         for (SelectOption option : ctrl.listSODonationFields) {
-            if (option.getValue() == UTIL_Namespace.alignSchemaNSWithEnvironment(
-                    'npsp__donationcampaignimported__c')) {
+
+            String paymentOriginIdField = SObjectType.DataImport__c.fields.Payment_Origin_Id__c.Name.toLowerCase();
+            if (option.getValue() == paymentOriginIdField) {
                 containsImportedField = true;
                 break;
             }
         }
         Test.startTest();
-        System.assert(containsImportedField, 'Help Text Mapping Enabled');
+        System.assert(!containsImportedField, 'Help Text Mapping Enabled');
```

Comment:
@callen-sfdo  Will this fail if the language is not english?

Reply:
@force2b No, these are just asserts not assertEquals, so its not doing any text comparison here.

Reply:
@callen-sfdo If I'm not mistaken, this text refers to the value of a [label](https://github.com/SalesforceFoundation/NPSP/blob/main/force-app/main/default/labels/CustomLabels.labels-meta.xml#L12426). When this test runs as the Spanish user in the automated Non-English build, it will fail because that label will have been [translated](https://github.com/SalesforceFoundation/NPSP/blob/main/force-app/main/default/translations/es.translation-meta.xml#L6330) into Spanish.

Reply:
@force2b Sorry, thought you were referring to the assert in particular.  In the comparison we are using .getValue() on the select option which gets the api value of the picklist option, and its comparing it to the api name of the payment origin field. So there should never be any translation concerns.

Reply:
@callen-sfdo Sorry. I completely misread the assert and thought it was basically `assertEquals`. The value of "Help Text Mapping Enabled" is the message displayed if the assert fails. I thought this was actually a label value, since there actually is a label with that exact value. So feel free to ignore this entirely.